src/main/steps/respondent/task-list/tasklist.ts

```diff
@@ -146,7 +146,7 @@ const getYourResponseSection = (sectionTitles, taskListItems, userCase: CaseWith
             id: 'respond_to_application',
             text: taskListItems.respond_to_application,
             status: getInternationalFactorsStatus(userCase),
-            href: !hasCitizenResponse ? URL.RESPOND_TO_APPLICATION + '/updateFlag' : null,
```

Comment:
We need this update flag endpoint, it is to update the flag informing us that user has started C7 response based on which we will display the train track state. Please revert this

Reply:
This was not working. Please check 622 PR.